packages/cli/src/lib/strings.ts

```diff
@@ -249,6 +254,7 @@ the PWA:
   promptKeystorePassword: 'Password for the Key Store:',
   promptKeyPassword: 'Password for the Key:',
   promptNewAppVersionName: 'versionName for the new App version:',
+  promptVersionCode: 'Starting Version code for the new app version',
```

Comment:
nit: Version should probably not be capitalized.
Also, for  consistency with the other prompts, could we have a colon at the end?

Reply:
Done